sdk/analytics/src/main/java/com/klaviyo/analytics/networking/KlaviyoApiClient.kt

```diff
@@ -241,13 +247,13 @@ internal object KlaviyoApiClient : ApiClient {
             while (apiQueue.isNotEmpty()) {
                 val request = apiQueue.poll()
 
-                when (request?.send()) {
-                    KlaviyoApiRequest.Status.Complete, KlaviyoApiRequest.Status.Failed -> {
+                when (request?.send { broadcastApiRequest(request) }) {
+                    Status.Complete, Status.Failed -> {
                         // On success or absolute failure, remove from queue and persistent store
                         Registry.dataStore.clear(request.uuid)
                         broadcastApiRequest(request)
```

Comment:
It looks like you broadcast above too...just checking if that's intentional.

Reply:
Yep! The one above is a pre-send broadcast, this one broadcasts the response code/body